docs/components/configurationPanel.md

```diff
@@ -12,9 +12,13 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
 
 # `gamepad.configurationPanel`
 
-This is a `viewComponent` and handles the state and functionality of the configuration panel. It also creates the
-configuration options and stores the custom gamepad configuration for use in the [`inputMapper`](inputMapper.md)
-component.
+This is a [`fluid.viewComponent`](https://tinyurl.com/y4u8xwpd) and creates the configuration options (Action Dropdown,
```

Comment:
"and creates" is odd, but rather than fix that, I'd suggest simplifying to "This `fluid.viewComponent` creates".